app/editor/src/features/content/papers/constants/defaultTotals.ts

```diff
@@ -0,0 +1,11 @@
+import { ITotalsInfo } from '../interfaces';
+
+export const defaultTotals: ITotalsInfo = {
+  topStories: 0,
+  commentary: 0,
+  featuredStories: 0,
+  published: 0,
+  publishedTopStories: 0,
```

Comment:
I'm not using these presently, but it may be useful to notice if content hasn't been published.